### ISSUE TITLE
Remove `master` branch and keep only cypress and baobab branches

### ISSUE THREAD

**Issue by martinkersner** [Bisonai/orakl#1164]

**Comment by jay-kim-kor** [comment]
@martinkersner I have a question for clarify. This issue apply to `orakl` repository. Right?
If it need apply to `orakl-helm-charts` repo, I'll transfer it.

**Comment by martinkersner** [comment]
@jay-kim-kor Yes, my bad. It should have been opened up in `orakl-helm-charts`. I am not sure it is possible to transfer issue between repositories. If not, let's close this one and open new one in `orakl-helm-charts`.
